apps/api/v1/pages/api/teams/[teamId]/_auth-middleware.ts

```diff
@@ -22,13 +22,13 @@ export async function checkPermissions(
   req: NextApiRequest,
   role: Prisma.MembershipWhereInput["role"] = MembershipRole.OWNER
 ) {
-  const { userId, isAdmin } = req;
+  const { userId, isSystemWideAdmin } = req;
   const { teamId } = schemaQueryTeamId.parse({
     teamId: req.query.teamId,
     version: req.query.version,
     apiKey: req.query.apiKey,
   });
-  return canUserAccessTeamWithRole(userId, isAdmin, teamId, role);
+  return canUserAccessTeamWithRole(userId, isSystemWideAdmin, teamId, role);
 }
 
 export async function canUserAccessTeamWithRole(
```

Comment:
should we change second param inside canUserAccessTeamWithRole from `isAdmin` to `isSystemWideAdmin`

Reply:
I don't think it is breaking, but we probably should for clarity and consistency!